app.go

```diff
@@ -21,7 +27,21 @@ func (a *App) startup(ctx context.Context) {
 	a.ctx = ctx
 }
 
-// Greet returns a greeting for the given name
-func (a *App) Greet(name string) string {
-	return fmt.Sprintf("Hello %s, It's show time!", name)
+// Export returns the number of transactions found at this address
+func (a *App) Export(addressOrEns string) string {
+	conn := rpc.NewConnection("mainnet", false, map[string]bool{})
+	addr, ok := conn.GetEnsAddress(addressOrEns)
```

Comment:
Does this work if I pass in an address?

Reply:
Yes. It works for both ENS names and addresses.

Reply:
In my basic test, looks like this didn't work with addresses. 

Additionally, the ENS name resolution seemed a bit wonky. 

Is there some way we can add tests in pure golang for this function?

Reply:
Tests added.